test/degreed/content/courses_test.rb

```diff
@@ -125,6 +124,89 @@ def test_get_courses_filtered_by_external_id
         assert_equal 200, response.code
         assert_equal "New Course", response.body.dig("data", "attributes", "title")
       end
+
+      def test_update_a_course_by_id
+        create_request = stub_request(:patch, "https://api.degreed.com/api/v2/content/courses/foo")
+          .with(
+            headers: {
+              "Authorization" => "Bearer someoauthtoken",
+              "content-type" => "application/json"
+            },
+            body: {
+              data: {
+                type: "content/courses",
+                id: "foo",
+                attributes: {
+                  title: "New Course Updated",
+                  obsolete: true
+                }
+              }
+            }.to_json
+          )
+          .to_return(
+            status: 200,
+            body: <<~JSON
+              {
+                "data": {
+                  "type": "content/courses",
+                  "id": "foo",
+                  "attributes": {
+                    "provider-code": null,
+                    "external-id": "arstaroisen",
+                    "degreed-url": "https://degreed.com/courses/?d=V9RVZY40PJ",
+                    "title": "New Course Updated",
+                    "summary": "A Summary Updated",
+                    "url": "https://dev.lessonly.com",
+                    "obsolete": false,
+                    "image-url": null,
+                    "language": null,
+                    "duration": 200,
+                    "duration-type": "Seconds",
+                    "cost-units": 0.0,
+                    "cost-unit-type": null,
+                    "format": null,
+                    "difficulty": null,
+                    "video-url": null,
+                    "created-at": "0001-01-01T00:00:00",
+                    "modified-at": "0001-01-01T00:00:00"
+                  },
+                  "links": {
+                    "self": "/content/courses/foo"
+                  }
+                }
+              }
+            JSON
+          )
+
+        response = Degreed::Content::Courses.new(token: "someoauthtoken").update(
+          id: "foo",
+          title: "New Course Updated",
+          obsolete: true
+        )
+
+        assert_requested create_request
+        assert_equal 200, response.code
+        assert_equal "New Course Updated", response.body.dig("data", "attributes", "title")
+      end
+
+      # def test_delete_course_by_id
```

Comment:
Yes, I think that's definitely appropriate. It will also make for an easier merge with the delete function PR. I got ahead of myself and forgot to remove this. 😬